e2e/__helpers__/test-case/processed-file-io.ts

```diff
@@ -1,12 +1,15 @@
+import { TransformOptions, TransformedSource } from '@jest/transform/build/types'
```

Comment:
Please send PRs for any types you need that are not exported at the top level of the module. Reaching into `build/` can break at any time

Reply:
do you meant FR and PRs ?

Reply:
Pull requests 🙂

Reply:
In this case, we already export `TransformOptions` at least. `TransformedSource` is not exported though (`ReturnType<Transformer['process']>` is clunky), so PR for that is welcome

https://www.runpkg.com/?@jest/transform@25.2.4/build/index.d.ts